tools/packaging/release/release.sh

```diff
@@ -47,6 +47,14 @@ function _check_required_env_var()
 		_die "\"$1\" environment variable is required but was not set"
 }
 
+function _create_new_release()
+{
+	_check_required_env_var "RELEASE_VERSION"
+	_check_required_env_var "GH_TOKEN"
+
+	gh release create $RELEASE_VERSION --generate-notes --title "Kata Containers $RELEASE_VERSION"
```

Comment:
I'm not sure if you want the `[ -z "$RELEASE_VERSION" ] && RELEASE_VERSION=$(cat "$repo_root_dir/VERSION")` code in this function for consistency with the others. Or maybe just extract that and run it at the start of the file outside of functions?

Reply:
Not for create release, as right now we're passing that env var, and I'm not sure what would be state of the repo on the next job of the same workflow after pushing the new version file.

Reply:
Yep - that's fair. After I reviewed this commit I saw the workflow with the `with`, but I thought it might be worth leaving the discussion about whether a refactor might be helpful